R/exposure_metrics.R

```diff
@@ -132,3 +132,71 @@ calc_auc_from_regimen <- function(
     target_design$type
   )
 }
+
+#' Calculate time to target attainment
+#'
+#' @param regimen PKPDsim regimen object
+#' @param target_design target design, created using `create_target_design()`
+#' @param auc_comp compartment to look for AUC
+#' @param model PKPDsim model object
+#' @param terminal_interval duration of final dosing interval
```

Comment:
This argument requires clearer documentation. Is this something that has to be specified by the user or can it be known based on target_type / regimen?